pkg/apis/core/v1/helper/helpers.go

```diff
@@ -85,6 +85,24 @@ func HugePageSizeFromResourceName(name v1.ResourceName) (resource.Quantity, erro
 	return resource.ParseQuantity(pageSize)
 }
 
+// HugePageUnitSizeFromByteSize returns hugepage size has the format.
+// `size` must be guaranteed to divisible into the largest units that can be expressed.
+// <size><unit-prefix>B (1024 = "1KB", 1048576 = "1MB", etc).
+func HugePageUnitSizeFromByteSize(size int64) (string, error) {
+	// Borrowed from opencontainers/runc/libcontainer/cgroups/utils.go
```

Comment:
clarify the algorithm isn't copied from there, just the list of units

Reply:
comment modification and commit squashing is done. thank you.